### DISCUSSION TITLE
How to delete data for OAuth2 token

### DISCUSSION THREAD

**Discussion by a-sobczyk** [thunderclient/thunder-client-support#1451] [Q&A]
I have a site for which I need to download a token which is different for each user. Unfortunately ThunderClient still only downloads the token for the first user I added. When I want to download a new token, there is no login page in the application, only the message "Success! Please close this tab." How to remove this data or how to change the user. In Postman there is an icon "clear cookies" which solves the issue. How to do it in ThubderClient?

**Comment by rangav** [comment]
Hi @a-sobczyk 

Are you using the free or paid version?

Generate the Token button again to clear previous token

**Reply by a-sobczyk** [reply]
We use the paid version. The problem has been solved. The browser had to be switched off because it was holding the chash login to the application.
And now the second question: Is it possible to set the option in tunderclient to open the browser window for OAtuh2 as a new incognito window?

**Reply by rangav** [reply]
Thanks for clarification 
Vscode controls the opening of new browser window. Don't think we have option for incognito. 

Will research and update here if there is a workaround
